src/C_API.jl

```diff
@@ -37,16 +43,21 @@ function _c_print(data::Ptr{Cvoid}, mode::Cint, msg::Ptr{Cchar})
         # mode & c_api_Output_Status == c_api_Output_Status ||
         mode & c_api_Output_Listing == c_api_Output_Listing
     )
-        io = unsafe_pointer_to_objref(data)::IO
-        print(io, unsafe_string(msg))
+        output_data = unsafe_pointer_to_objref(data)::OutputData
+        if !output_data.silent
+            print(stdout, unsafe_string(msg))
+    end
     end
     return
 end
 
-function OutputInterface(io)
+function OutputInterface(output_data)
     _C_PRINT = @cfunction(_c_print, Cvoid, (Ptr{Cvoid}, Cint, Ptr{Cchar}))
     _C_FLUSH = @cfunction(_c_flush, Cvoid, (Ptr{Cvoid}, Cint))
-    return OutputInterface(pointer_from_objref(io), _C_PRINT, _C_FLUSH)
+    # Without this call to flush everything segfaults on Windows. Very unclear
+    # why, though...
+    flush(stdout)
```

Comment:
I _really_ don't understand why things work with this `flush` call, but segfault with it... That does make me a bit nervous that I'm missing something crucial here? If someone has an insight what might be going on, it would be great!

Reply:
No idea... How did you even figure out that this stopped the segfault!